pkg/types/types.go

```diff
@@ -243,6 +243,26 @@ type Version struct {
 	Version string `json:"version"`
 }
 
+type NodeStatisticsResources struct {
+	CRU   int `json:"cru"`
+	HRU   int `json:"hru"`
+	IPV4U int `json:"ipv4u"`
+	MRU   int `json:"mru"`
+	SRU   int `json:"sru"`
+}
+
+type NodeStatisticsUsers struct {
```

Comment:
I don't understand this struct relation to Users?

Reply:
That's the name of the field in the response to `zos.statistics.get`